stratum/hal/lib/common/config_monitoring_service.cc

```diff
@@ -266,10 +269,17 @@ ::grpc::Status ConfigMonitoringService::DoSet(::grpc::ServerContext* context,
                             status.error_message());
     }
 
+#ifndef P4OVS_CHANGES
```

Comment:
This seems common code. Why are we commenting it out?

Reply:
I don't know. That's why it's conditionalized with `P4OVS_CHANGES`.

Reply:
Lets remove it then. We can figure this out with the openconfig changes that will follow.

Reply:
Do you mean remove the conditional, or remove the code?

I'm not going to remove the code. We'd be making a (possibly breaking) change to a common file without being able to say why we're doing it.

If you mean remove the conditional, why? It does no harm if P4OVS_CHANGES is disabled, and the P4-OVS change was presumably made for a reason (even if we don't know what it is right now).